backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.MappedSuperclass;
+
+import org.hibernate.annotations.Type;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+
+@MappedSuperclass
+public abstract class BaseTestInfo extends AuditedEntity {
+
+	@ManyToOne(optional = false)
+	@JoinColumn(name = "patient_id", updatable = false)
+	private Person patient;
+
+	@ManyToOne(optional = false)
+	@JoinColumn(name = "organization_id", updatable = false)
+	private Organization organization;
+
+	@ManyToOne(optional = false)
+	@JoinColumn(name = "device_type_id")
```

Comment:
This is the place where the common base class gets a little rickety: for TestEvent it's immutable—but then, TestEvent itself is immutable, so that's no surprise. For TestOrder, this is _not_ immutable, because until the test is given, they might change which device they're running it on. So if the two classes are going to share this attribute, it has to be updatable. (If we get to where TestOrder wants it to be optional and updatable, while TestEvent wants it to be neither, we probably just pull it down into the concrete classes.)